lib/bolt/config/options.rb

```diff
@@ -172,6 +183,18 @@ module Options
           _example: "~/.puppetlabs/bolt/inventory.yaml",
           _default: "project/inventory.yaml"
         },
+        "plugin-cache" => {
+          description: "This feature is experimental. Enable plugin caching and set the time-to-live.",
+          type: Hash,
+          _plugin: false,
+          _example: { "ttl" => 3600 },
+          properties: {
+            "ttl" => {
+              description: "Time in seconds to keep the plugin cache.",
+              type: Integer
+            }
+          }
```

Comment:
```suggestion
          required: ["ttl"],
          properties: {
            "ttl" => {
              description: "Time in seconds to keep the plugin cache.",
              type: Integer
            }
          },
          _plugin: false,
          _example: { "ttl" => 3600 }
```

Make sure `ttl` is required & keeping key order consistent between options.